control/installation.SLES.xml

```diff
@@ -35,66 +35,117 @@ defined by a role can be overridden in the next steps if necessary.&lt;/p&gt;</l
     </texts>
 
     <partitioning>
-        <try_separate_home config:type="boolean">true</try_separate_home>
-        <limit_try_home>5GB</limit_try_home>
-        <root_space_percent>40</root_space_percent>
-        <root_base_size>3GB</root_base_size>
-        <root_max_size>10GB</root_max_size>
-        <proposal_lvm config:type="boolean">false</proposal_lvm>
-        <vm_desired_size>15GB</vm_desired_size>
-        <vm_home_max_size>25GB</vm_home_max_size>
-        <btrfs_increase_percentage config:type="integer">300</btrfs_increase_percentage>
-        <!-- the default subvolume "@" was requested by product management -->
-        <btrfs_default_subvolume>@</btrfs_default_subvolume>
-
-        <!-- Subvolumes to be created for a Btrfs root file system -->
-        <!-- copy_on_write is true by default -->
-        <!-- The XML parser is very simplistic, thus not using attributes, but sub-elements -->
-        <subvolumes config:type="list">
-            <subvolume>
-                <path>home</path>
-            </subvolume>
-            <subvolume>
-                <path>opt</path>
-            </subvolume>
-            <subvolume>
-                <path>srv</path>
-            </subvolume>
-            <subvolume>
-                <path>tmp</path>
-            </subvolume>
-            <subvolume>
-                <path>usr/local</path>
-            </subvolume>
-            <!-- unified var subvolume - https://lists.opensuse.org/opensuse-packaging/2017-11/msg00017.html -->
-            <subvolume>
-                <path>var</path>
-                <copy_on_write config:type="boolean">false</copy_on_write>
-            </subvolume>
-
-            <!-- architecture specific subvolumes -->
-
-            <subvolume>
-                <path>boot/grub2/i386-pc</path>
-                <archs>i386,x86_64</archs>
-            </subvolume>
-            <subvolume>
-                <path>boot/grub2/x86_64-efi</path>
-                <archs>x86_64</archs>
-            </subvolume>
-            <subvolume>
-                <path>boot/grub2/powerpc-ieee1275</path>
-                <archs>ppc,!board_powernv</archs>
-            </subvolume>
-            <subvolume>
-                <path>boot/grub2/x86_64-efi</path>
-                <archs>x86_64</archs>
-            </subvolume>
-            <subvolume>
-                <path>boot/grub2/s390x-emu</path>
-                <archs>s390</archs>
-            </subvolume>
-        </subvolumes>
+       <expert_partitioner_warning config:type="boolean">false</expert_partitioner_warning>
```

Comment:
this is new entry? I do not see any counter part in old config

Reply:
https://github.com/yast/yast-storage-ng/blob/master/doc/old_and_new_proposal.md#other-general-options

Reply:
@jsrain fine for me, but I would like to have at least in comment rationale *why* we disable this warning for SLES.

Reply:
Yes, I've put in a few settings even with default value so people learn what's there.

Reply:
@jreidinger: doc says default is `false`.
Hm, it's still not even implemented. :-/

Reply:
@wfeldt OK, I was just curious why we stop showing expert partitioner warning and it will be default now. jsrain already approve it, so fine for me.